deploy/helm-operator-crds.yaml

```diff
@@ -0,0 +1,355 @@
+                  type: boolean
+                enable:
+                  description: Enable will mark this Helm release for rollbacks.
+                  type: boolean
+                force:
+                  description: Force will mark this Helm release to `--force` rollbacks.
+                    This forces the resource updates through delete/recreate if needed.
+                  type: boolean
+                maxRetries:
+                  description: MaxRetries is the maximum amount of upgrade retries
+                    the operator should make before bailing.
+                  type: integer
+                  format: int64
+                recreate:
+                  description: Recreate will mark this Helm release to `--recreate-pods`
+                    for if applicable. This performs pod restarts.
+                  type: boolean
+                retry:
+                  description: Retry will mark this Helm release for upgrade retries
+                    after a rollback.
+                  type: boolean
+                timeout:
+                  description: Timeout is the time to wait for any individual Kubernetes
+                    operation (like Jobs for hooks) during rollback.
+                  type: integer
+                  format: int64
+                wait:
+                  description: Wait will mark this Helm release to wait until all
+                    Pods, PVCs, Services, and minimum number of Pods of a Deployment,
+                    StatefulSet, or ReplicaSet are in a ready state before marking
+                    the release as successful.
+                  type: boolean
+            skipCRDs:
+              description: SkipCRDs will mark this Helm release to skip the creation
+                of CRDs during a Helm 3 installation.
+              type: boolean
+            targetNamespace:
+              description: TargetNamespace overrides the targeted namespace for the
+                Helm release. The default namespace equals to the namespace of the
+                HelmRelease resource.
+              type: string
+            timeout:
+              description: Timeout is the time to wait for any individual Kubernetes
+                operation (like Jobs for hooks) during installation and upgrade operations.
+              type: integer
+              format: int64
+            valueFileSecrets:
+              description: ValueFileSecrets holds the local name references to secrets.
+                DEPRECATED, use ValuesFrom.secretKeyRef instead.
+              type: array
+              items:
+                type: object
+                required:
+                - name
+                properties:
+                  name:
+                    type: string
+            values:
+              description: Values holds the values for this Helm release.
+              type: object
+              x-kubernetes-preserve-unknown-fields: true
```

Comment:
This will break compatibility with Kubernetes < 1.17, people will have to disable kubectl validation to apply it.

Reply:
It works for Kubernetes 1.15 and above, but fair point for our beloved GKE/EKS users. I have removed the flag putting it there.